admin/webapp/websrc/app/common/utils/auth.utils.ts

```diff
@@ -82,6 +82,7 @@ export class AuthUtilsService {
 
   private getCacheUserPermission() {
     if (
+      GlobalVariable.isRemote ||
```

Comment:
Is there any effect when GlobalVariable.isRemote is true or false in this condition with "or" logic?

Reply:
when isRemote is true, the login user switches to the managed cluster. In this case, the remote_global_permissions needs to be updated.